ui/raidboss/data/03-hw/raid/a8n.js

```diff
@@ -0,0 +1,518 @@
+'use strict';
+
+// ALEXANDER - THE BURDEN OF THE SON NORMAL
+// A8N
+
+[{
+  zoneId: ZoneId.AlexanderTheBurdenOfTheSon,
+  timelineFile: 'a8n.txt',
+  timelineTriggers: [
+    {
+      id: 'A8N Hydrothermal Missile',
+      regex: /Hydrothermal Missile/,
+      beforeSeconds: 3,
+      response: Responses.tankCleave('info'),
+    },
+    {
+      id: 'A8N Flarethrower',
+      regex: /Flarethrower/,
+      beforeSeconds: 3,
+      response: Responses.tankCleave('info'),
+    },
+    {
+      id: 'A8N Super Jump Soon',
+      regex: /Super Jump/,
+      beforeSeconds: 8,
+      infoText: {
+        en: 'Bait Super Jump',
+        de: 'Supersprung ködern',
+        fr: 'Attirez le Super saut',
+        ja: 'スーパージャンプを誘導',
+        cn: '引导超级跳',
+        ko: '슈퍼 점프',
+      },
+    },
+  ],
+  triggers: [
+    {
+      id: 'A8N Megabeam Onslaughter',
+      netRegex: NetRegexes.startsUsing({ source: 'Onslaughter', id: '1732', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Schlachter', id: '1732', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Attaqueur', id: '1732', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'オンスローター', id: '1732', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ source: '突击者', id: '1732', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ source: '맹습자', id: '1732', capture: false }),
+      // Insert sound effect from Arthars here.
```

Comment:
Nice copy and paste :P

Reply:
It seemed appropriate to leave in even if I'm not Arthars' biggest fan.